### ISSUE TITLE
Discord ban

### ISSUE THREAD

**Issue by HAMMERxPL** [ViaVersion/ViaVersion#2871]
I got banned from your discord server without any warning, I notice that you are full of negative energy there, but if you want to ban at least give the warning or keep your negative feelings for yourself. It's pretty disappointing that you provide such a popular plugin, but you ignore hints or questions, leaving them with no help, answers like "remove viaversion" or poor performance that can lead to low general smoothness of the whole server.

### Steps to Reproduce

1. Try to help and improve ViaVersion.
2. Get banned.

**Comment by Barvalg** [comment]
![Error: Invalid or missing dump](https://img.shields.io/badge/Error-Invalid/missing%20dump-red?style=flat-square) </br>

**Comment by Kichura** [comment]
We have requested you many times to provide your viaversion dump(s) to #support which you have failed to do so,

Followed up by you constantly asking us to improve via* just because the hardware CPU or whatever's value increased by a tiny amount higher,

and to top that off; You also failed to follow instructions that we have clearly explained from the get-go including all kinds of suggestions.

In conclusion: The ban is fully valid, and we are not removing it anytime soon.

**Comment by HAMMERxPL** [comment]
> We have requested you many times to provide your viaversion dump(s) to #support which you have failed to do so,
> 
> Followed up by you constantly asking us to improve via* just because the hardware CPU or whatever's value increased by a tiny amount higher,
> 
> and to top that off; You also failed to follow instructions that we have clearly explained from the get-go including all kinds of suggestions.
> 
> In conclusion: The ban is fully valid, and we are not removing it anytime soon.

I was NEVER (or in pretty long time) asked for the viaversion dump, as my issues were unrelated to the server's specifications.

Your only suggestion that I have read was "remove viaversion". 20% of cpu time is not "a tiny value". I asked for using reply function and when I checked again I was banned, I didn't have chance to get the instructions, if there were any. 

I don't know why are you so negative about my person, if that's possible you can pass me your discord tag, so we can talk about it privately to avoid spamming.
